Texstyle/TexstyleTests/LineAttributesTests.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright © 2019 Rosberry. All rights reserved.
+//
+
+import XCTest
+@testable import Texstyle
+
+final class LineAttributesTests: XCTestCase {
+
+    func testCopy() {
+        //Given
+        let attributes = LineAttributes.random
+        //When
+        let copy = attributes.copy()
+        //Then
+        XCTAssertEqual(copy, attributes, "Copy is not equal to line attributes")
```

Comment:
> "Copy is not equal to line attributes"

NOT equal?

Reply:
I think so, this message contains a description what was wrong in this assertion